### PR TITLE
Fix #110896 fashionunited.com

### LINKED ISSUE #110896: fashionunited.com
### Issue URL (Ads)
 [https://fashionunited.com](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Ffashionunited.com)
### Comment
Username: @Morku
> Several Ad placeholder

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>
<details>
  <summary>Screenshot 2</summary>

  Screenshot 2
</details>
<details>
  <summary>Screenshot 3</summary>

  Screenshot 3
</details>

### System configuration

Information | value
--- | ---
AdGuard product: | AdGuard for Windows v7.8
Browser: | Firefox
AdGuard driver: | WFP
Stealth mode options: | Hide your search queries, <br>Send Do-Not-Track header, <br>Remove X-Client-Data header from HTTP requests, <br>Strip URLs from tracking parameters
DNS filtering: | server: `Cloudflare DNS-over-HTTPS`<br>filters: `"AdGuard DNS filter" with AdGuard Home`
Filters: | <b>Ad Blocking:</b><br/>AdGuard Mobile Ads, <br/>AdGuard Base<br/><br/><b>Privacy:</b><br/>AdGuard Tracking Protection, <br/>AdGuard URL Tracking<br/><br/><b>Social Widgets:</b><br/>AdGuard Social Media<br/><br/><b>Annoyances:</b><br/>AdGuard Annoyances<br/><br/><b>Language-specific:</b><br/>AdGuard German
Userscripts: | https://userscripts.adtidy.org/release/adguard-extra/1.0/adguard-extra.user.js

### PR DESCRIPTION
**Pull request by Morku**
Fix #110896

Page exist in multiple language, e.g. fashionunited.es, fashionunited.fr, fashionunited.com, fashionunited.ch and so on, so I made it more general.

### CODE REVIEW

**Review by Yuki2718 — CHANGES_REQUESTED** [review]
Better `fashionunited.*##.adunitContainer:upward(3)`
and add the domain to `ahaan.co.uk,afteracademy.com,shoppingcidademaringa.com.br##.MuiSnackbar-anchorOriginBottomCenter` in cookies-specific.txt

**Comment by Morku** [comment]
@Yuki2718 
Rule works fine on Homepage, but will leave a placeholder on https://fashionunited.com/just-in (Screenshot 3 of my report). Thats why I created the second rule with `class^="MuiGrid-root css-".` Any idea how to block both placeholders with one rule?

**Comment by Yuki2718** [comment]
Well, actually at first I saw them so suggested `fashionunited.*##.MuiGrid-root > div.MuiGrid-root[padding] > div[class^="css"] > div > div[class^="css"] > div.adunitContainer:upward(4)` as well before editing my comment but then didn't see - now I see agian, strange.

**Review by Morku — COMMENTED** [review]
update

**Review by Yuki2718 — APPROVED** [review]

File: EnglishFilter/sections/specific.txt [hunk 1]
```diff
@@ -294,6 +294,7 @@ linkjust.com,updatesmarugujarat.in,mexashare.com,links4u.me,linku.us,skips.link,
 !
 !
 !
+fashionunited.*##.adunitContainer:upward(3)
```
Comment (zloyden) [inline EnglishFilter/sections/specific.txt:297]: @Morku It has to be in `css_extended` file and written via `#?#` - will move it.
As I can see, it works fine on the website.
Btw, we have added comments at the headers of the each file - please read them.

**Review by zloyden — CHANGES_REQUESTED** [review]

**Review by zloyden — APPROVED** [review]

**Pull request merged**